<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="main_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>open file</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ai_chat">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>-1</y>
      <width>231</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>AI chat</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="ai_v_layout">
      <item>
       <widget class="QTextEdit" name="ai_history"/>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="ai_request">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>231</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ai_send_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>SEND</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="save_file">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>save file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
